RAW/RawModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  RawModel.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import Foundation
+
+struct RawModel: Codable {
+    
+    let userid : Int?
+    let id : Int?
```

Comment:
why are you making all of these values optional?

Reply:
otherwise I have to initialise the values here.